src/runtime/pay-client.js

```diff
@@ -104,11 +115,13 @@ export class PayClient {
    * @return {!PaymentsAsyncClient}
    * @private
    */
-  createClient_(options, googleTransactionId, handler) {
+  async createClient_(options, googleTransactionId, handler) {
+    await PayReadyPromise;
     // Assign Google Transaction ID to PaymentsAsyncClient.googleTransactionId_
     // so it can be passed to gpay_async.js and stored in payment clearcut log.
-    PaymentsAsyncClient.googleTransactionId_ = googleTransactionId;
-    return new PaymentsAsyncClient(
+    google.payments.api.PaymentsAsyncClient.googleTransactionId_ =
```

Comment:
We might need to quote the `googleTransactionId_` property name, so Terser doesn't mangle it in the new Vite build system